tasks/test_prep.yml

```diff
@@ -120,13 +120,14 @@
       pkg_testing_path: "{{ pkg_testing_path_output.stdout }}"
     when: lookup('env', 'USE_LOCAL_PKG_TEST') == "1"
 
-  - name: copy package-testing directory to remote
-    synchronize:
-      src: "{{ pkg_testing_path }}"
-      dest: /
-      recursive: yes
-      perms: yes
-    when: lookup('env', 'USE_LOCAL_PKG_TEST') == "1"
+# Disabled test as jobs are failing on this step for OL8 due to synchronize module not present in OL8 ansible-core modulei and this test is always skipped as we don't use 'USE_LOCAL_PKG_TEST'
+    #  - name: copy package-testing directory to remote
+    #    synchronize:
+    #      src: "{{ pkg_testing_path }}"
```

Comment:
If this test does not run anywhere, why to keep it commented. Will it not make sense to remove it ?

Reply:
What about other platforms where this test can run?

Reply:
lets disable it for a month or so , in the meantime , all jobs can be executed and if we don't get any issues with any jobs , i can remove it , does it sounds good ?

Reply:
Commenting the test will disable the test on all platforms. Better to disable it on OL8 only. If the test does not run on any platform we need to either find the reason why it is skipped on other platforms or remove the test in case requirements are not met